stable/velero/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.11.0
 description: A Helm chart for velero
 name: velero
-version: 1.0.0
+version: 1.1.0
```

Sorry I didn't catch this before. Documentation-only changes should just be a patch version bump. Due to the issues with CI I would be okay merging as is for now but I would like to get a second opinion first.

```suggestion
version: 1.0.1
```

---

Agreed. Let's bump the patch version.